package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.5",
     "react-scripts": "4.0.3",
```

Comment:
- [ ] It would be better to add example books to the initial state object
![Screenshot from 2021-10-19 16-26-44](https://user-images.githubusercontent.com/46388113/137952597-e756ac47-9a68-4641-a8e5-569280825b31.png)